lib/livebook/runtime/evaluator/doctests.ex

```diff
@@ -23,7 +23,9 @@ defmodule Livebook.Runtime.Evaluator.Doctests do
             |> Enum.map(&run_test/1)
```

Comment:
I wonder if we could stream the gutter results per test? Before we run the test, we add a gray dot, after we run it, we make it red or green?

Reply:
Oh so that way the tests update the indicator as they are ran on the server, super cool!

This is what it looks like :)


https://github.com/livebook-dev/livebook/assets/7144046/00d5bdb5-558c-4ec0-9a61-1e9876e09c39